src/libsystemd/sd-bus/bus-common-errors.c

```diff
@@ -59,6 +59,9 @@ BUS_ERROR_MAP_ELF_REGISTER const sd_bus_error_map bus_common_errors[] = {
         SD_BUS_ERROR_MAP(BUS_ERROR_SLEEP_VERB_NOT_SUPPORTED,     EOPNOTSUPP),
         SD_BUS_ERROR_MAP(BUS_ERROR_SESSION_BUSY,                 EBUSY),
         SD_BUS_ERROR_MAP(BUS_ERROR_NOT_YOUR_DEVICE,              EPERM),
+        // needs to be EOPNOTSUPP for proper handling in callers of logind_schedule_shutdown()
```

Comment:
`//` we use for FIXME style comments only, not for descriptive comments see https://systemd.io/CODING_STYLE/#formatting